### ISSUE TITLE
Hyperparameter tuning

### ISSUE THREAD

**Issue by goutamyg** [researchmm/Stark#107]
Hi! Thank you for publishing your code.

In your paper, you have mentioned various training and model-related choices/hyperparameters (e.g., optimizer, scheduler, number of transformer heads, number of encoder-decoder layers, L_1, and GIOU weightages). Can you please suggest which dataset was used to tune these hyperparameter values? Or was it fine-tuned using the test-set itself?
